frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  TextContent,
+  Text,
+  TextVariants,
+  Card,
+  CardBody,
+  AlertVariant,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import {
+  GreenCheckCircleIcon,
+  YellowExclamationTriangleIcon,
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@console/shared/src';
+import { humanizeBinaryBytes } from '@console/internal/components/utils';
+import { MINIMUM_NODES, NO_PROVISIONER } from '../../../../constants';
+import {
+  capacityAndNodesValidate,
+  getAllZone,
+  getTotalCpu,
+  getTotalMemory,
+} from '../../../../utils/create-storage-system';
+import { OSD_CAPACITY_SIZES, TotalCapacityText } from '../../../../utils/osd-size-dropdown';
+import {
+  ValidationMessage,
+  VALIDATIONS,
+  ValidationType,
+} from '../../../../utils/common-ocs-install-el';
+import { WizardNodeState, WizardState } from '../../reducer';
+import './review-and-create-step.scss';
+
+const NodesCard: React.FC<NodeCardProps> = ({ nodes }) =>
+  !!nodes.length && (
+    <Card isCompact isFlat component="div" className=".odf-review-and-create__card">
+      <CardBody isFilled className=".odf-review-and-create__card-body">
+        <TextContent>
+          {nodes.map((node) => (
+            <Text key={node.name} component={TextVariants.small}>
+              {node.name}
+            </Text>
+          ))}
+        </TextContent>
+      </CardBody>
+    </Card>
+  );
+
+type NodeCardProps = { nodes: WizardNodeState[] };
+
+const REVIEW_ICON_MAP = {
+  [AlertVariant.success]: GreenCheckCircleIcon,
+  [AlertVariant.warning]: YellowExclamationTriangleIcon,
+  [AlertVariant.info]: BlueInfoCircleIcon,
+  [AlertVariant.danger]: RedExclamationCircleIcon,
+};
+
+export const ReviewListTitle: React.FC<ReviewListTitleProps> = ({ text }) => (
+  <dt>
+    <TextContent>
+      <Text component={TextVariants.h4}>{text}</Text>
+    </TextContent>
+  </dt>
+);
+
+type ReviewListTitleProps = { text: string };
+
+export const ReviewListBody: React.FC<ReviewListBodyProps> = ({
```

Comment:
reuse from frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/review-and-create.tsx

Reply:
same as above

Reply:
ack